roofit/histfactory/inc/RooStats/HistFactory/Data.h

```diff
@@ -30,23 +30,24 @@ class Data {
   /// constructor from name, file and path. Name of the histogram should not include the path
   Data( std::string HistoName, std::string InputFile, std::string HistoPath="" );
 
-  std::string GetName() { return fName; }
+  std::string GetName() const { return fName; }
```

Comment:
It is not maybe better to  return here and similar below a const reference to std::string ?

Reply:
Better for sure, but this didn't do any harm in the past. The real CPU time is spent getting all histograms from files and converting them to likelihood models.